lib/awful/menu.lua

```diff
@@ -541,6 +541,16 @@ function menu.clients(args, item_args, filter)
     return m
 end
 
+local clients_menu = nil
```

Comment:
i think it should be `menu.clients_menu_instance = nil` here instead to expose it

however i am not sure if we need to move it out from rc.lua, let's wait for more opinions

Reply:
Works for me without being exposed and I can't see a need to access it elsewhere but I'm open to anything

My reasoning for moving it out of rc.lua is simply that AFAICT there should be no reason for someone to need to edit it (except the "theme" part which would remain in rc.lua anyway in the call)